### PR TITLE
webui: Fix WebUI end2end tests after recent renaming and introducing of screenshot limit

### PR DESCRIPTION
**Pull request by velezd**
Looks like we missed some things in the recent changes of WebUI tests. I guess nobody tried to run them.

### CODE REVIEW

**Comment by velezd** [comment]
There seem to be more issues with the end2end tests, I will look into it next week.

**Comment by velezd** [comment]
The last one was issue with my setup.

File: ui/webui/test/end2end/default.tc.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ priority: 9
 execution:
   type: anaconda-webui
   automation_data:
-    script_file: ./ui/webui/test/integration/default.py
+    script_file: ./ui/webui/test/end2end//default.py
```
Comment (KKoukiou) [inline ui/webui/test/end2end/default.tc.yaml:11]: Double //

**Review by KKoukiou — APPROVED** [review]

**Review by KKoukiou — COMMENTED** [review]
I think in anaconda they have this convention that commit changing infra files like .github directory, need the infra prefix in the commit message. that's why the first test is failing for this PR. can you please adjust for the test to pass?

**Comment by velezd** [comment]
> I think in anaconda they have this convention that commit changing infra files like .github directory, need the infra prefix in the commit message. that's why the first test is failing for this PR. can you please adjust for the test to pass?

I've changed it but the check doesn't seem to be working correctly, or I'm missing something.
`infra: webui: Fix end2end tests after renaming --> Is missing 'infra:' prefix or '(#infra)' postfix`

**Comment by KKoukiou** [comment]
@velezd it's because the check is very naive, and wants pRs with 'infrastructure label' to have ALL commits starting with infra which is wrong. I suggest split the PR into 2 PRs, one with the infra commit and one without.

**Review by jkonecny12 — APPROVED** [review]
Looks good to me. I think it's fine to ignore the infra check here.

**Pull request merged**